### ISSUE TITLE
Why the result of TSM-Mobilenetv2 in kinectcis doesnt reach the TSM official Repo? 

### ISSUE THREAD

**Issue by barbecacov** [open-mmlab/mmaction2#1016]
The result of [TSM offical repo](https://github.com/mit-han-lab/temporal-shift-module) is **69.5%**.

But in Mmaction2 the result is **68.46%**.

And I compare the detalis of thraining with TSM's.    Nearly the same。So i wonder whethear you know the difference about the details of training or the model architecture which leads to the gap。

Thank you very much!

**Comment by irvingzhang0512** [comment]
Actually, TSM official repo hasn't share the hyper params(such as learning rate, weight decay, batch size, etc.) of training tsm-mobilentv2.
